### PR TITLE
oh no can't auto merge

### PR DESCRIPTION
**Pull request by briarsychung**

### CODE REVIEW

**Comment by briarsychung** [comment]
Ack @SheepTester help idk what how which to do this!!!

**Comment by briarsychung** [comment]
I can resolve `cocnept-designs/v2/index.html` but the other two idk what your edits mean.

**Comment by SheepTester** [comment]
lol

File: school/school.js [hunk 1]
```diff
@@ -103,7 +116,7 @@ async function generateSchool() {
     for (let iterations = 0; iterations < MAX_ITER; iterations++) {
         for (let i = 0; i < Students.length; i++) {
             let student = Students[i];
-
+\
```
Comment (SheepTester) [inline school/school.js:119]: oops my bad

**Review by SheepTester — COMMENTED** [review]
accidentally left a backslash when resolving conflict oops my toolbox is leaky

**Comment by briarsychung** [comment]
Thank

**Pull request merged**